pkg/accounting/accounting.go

```diff
@@ -439,7 +439,7 @@ func (c *creditAction) Cleanup() {
 	defer c.accountingPeer.lock.Unlock()
 
 	if c.price.Cmp(c.accountingPeer.reservedBalance) > 0 {
-		c.accounting.logger.Error(nil, "attempting to release more balance than was reserved for peer")
+		c.accounting.logger.Error(nil, "allowance expectation refused", "peer_address", c.peer)
```

Comment:
why can't we go with "attempting to release more balance than was reserved for peer"